### ISSUE TITLE
Build Failing for 6 months

### ISSUE THREAD

**Issue by samuelcolvin** [niwinz/django-jinja#146]
I'm a big fan of django-jinja but I'm concerned to see that build has been failing for 6 months and there hasn't been any development.

Is the project still be developed/maintained? Will it work with dj 1.9?

I'm just about to start a new django package and was intending to use django-jinja but I don't want to rely on a package which will fade away imminently.

**Comment by niwinz** [comment]
Hi @samuelcolvin 
The package is alive and used in production on my own projects and projects of my company. This not will be changed.

Lamentable the last months I was so busy in other stuff and I don't have much time for "minor issues", the package as is works properly and does not need much maintenance. About failing tests you are right, this is a very minor issue and simply forgot about it.

If you found an important bug it will be addressed much quickly...
And if you have improvements, any help is very welcome.

**Comment by samuelcolvin** [comment]
Great, good to hear.

We're using django-jinja in production ourselves at my company too, I just wanted to check going forward

**Comment by niwinz** [comment]
As I answered in other issue, I have plans to drop support for older versions in future versions of django jinja. Because the current version tries to backpass all the peculiarity of django <= 1.6, django == 1.7 and django >= 1.8 and the code becomes ugly and unmaintenable...

That do you think about?

**Comment by samuelcolvin** [comment]
Personally we moved to 1.8 more or less as soon as it was released.

The only over version of django currently supported is 1.7 which [expires in december](https://www.djangoproject.com/download/#supported-versions).

If people are still running on older versions of django they can stick with django-jinja 1.4.1 which is tried and tested and is unlikely to suddenly exhibit a serious flaw.

I say drop support for everything < 1.8.
